roboplot/svg/svg_parsing.py

```diff
@@ -0,0 +1,142 @@
+import itertools
+import re
+import warnings
+
+import numpy as np
+import svgpathtools as svg
+
+from roboplot.core.curves import Curve
+
+
+def parse(filepath: str):
+    """
+    Parses an svg file to return an iterable of SVGPaths.
+
+    Args:
+        filepath (str): curve to the svg file.
+
+    Returns:
+        an iterable of SVGPath objects.
+
+    """
+    paths, _, svg_attributes = svg.svg2paths2(filepath)
+    setsofsubpaths = [p.continuous_subpaths() for p in paths]
+    paths = itertools.chain.from_iterable(setsofsubpaths)
+
+    svg_attributes = SvgAttributes(svg_attributes)
```

Comment:
Fixed :)
But I won't merge the pull request yet since I noticed you've branched off M_MotorFunctionality